src/browser-connection/index.js

```diff
@@ -35,7 +35,7 @@ export default class BrowserConnection extends EventEmitter {
         this.idleUrl      = `${gateway.domain}/browser/idle/${this.id}`;
         this.statusUrl    = `${gateway.domain}/browser/status/${this.id}`;
 
-        this.on('error', () => this.close());
+        this.on('error', async () => await this.close());
```

Comment:
This `await` here doesn't make sense, because event hanlder will be executed in it's own execution context. We